### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by rajeshr-ind** [pathikrit/better-files#657] [Ideas]
hi, I find the readme file lengthy. Shall we move it to wikis ?

**Comment by pathikrit** [comment]
wiki is non-standard and Github specific. If tomorrow, I decided to move from Github to Sourcegraph/Gitlabs/youtrack/whatever new comes, I can't simply reimport the git repo.

In my 20-year career as an engineer, I have already moved my code 3 times whatever was trendy - from Sourceforce to Google Code to Github.

**Comment by rajeshr-ind** [comment]
okay @pathikrit, thanks for the info.
